pkg/core/xds/resource.go

```diff
@@ -167,3 +170,20 @@ func (s *ResourceSet) List() ResourceList {
 
 	return list
 }
+
+func (s *ResourceSet) GetByOwner() map[rules.UniqueResourceIdentifier]map[string][]*Resource {
```

Comment:
nit: the name sort of means that `owner` is passed as an argument and we get values for the `owner` (i.e. we have `GetByKey(key)` or `GetByMesh(mesh)` in the code). This method should be `IndexByOwner()`

Reply:
changed